### PR TITLE
add bit-op docs for nucleotides

### PR DESCRIPTION
**Pull request by bicycle1885**

### CODE REVIEW

File: docs/src/man/seq.md [hunk 1]
```diff
@@ -91,6 +91,45 @@ true
 
 ```
 
+Every nucleotide is encoded using lower 4 bits of a byte. An unambiguous
+nucleotide has only one *true* bit and the other bits are *false*s. The table
```
Comment (Ward9250) [inline docs/src/man/seq.md]: "using lower 4 bits" <- "using the lower 4 bits".

Also should it be "has only one set bit" & "the other bits are unset"? I'm not sure which is more intuitive to less computer savvy biologists,

File: docs/src/man/seq.md [hunk 2]
```diff
@@ -91,6 +91,45 @@ true
...
+below summarizes all unambiguous nucleotides and their corresponding bits. An
+ambiguous nucleotide is the bitwise OR of unambiguous nucleotides that the
+ambiguous nucleotide can take. For example, `DNA_R`--`DNA_A` or `DNA_G`--is
```
Comment (Ward9250) [inline docs/src/man/seq.md]: Suggestion: "`DNA_R` (meaning the nucleotide is either `DNA_A` *or* `DNA_G`)" ?

**Comment by Ward9250** [comment]
Two minor comments for what is written but mostly just suggestions - looks good!

File: docs/src/man/seq.md (hunk 1 above)
Comment (bicycle1885) [inline docs/src/man/seq.md]: Thanks again!

That sounds more intuitive considering bit flags.

File: docs/src/man/seq.md (hunk 2 above)
Comment (bicycle1885) [inline docs/src/man/seq.md]: 👍

**Pull request merged**